### ISSUE TITLE
Bacterial infection

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6193]
**Preferred term label:**
Bacterial infection

**Synonyms**
Bacterial infections

**Definition (free text, please give PubMed ID)**
An infection caused by bacteria.
PMCID: PMC7149789

**Parent term (use hpo.jax.org/app)**
Unusual infection
-> "Bacterial infection" as parent term of "Recurrent bacterial infections"

**Comment by pnrobinson** [comment]
A single bacterial infection is a disease. If a person has recurrent bacterial infections, then the HPO regards this as a phenotypic feature (this medical history) because it can point to an immune defect. For the single episode, we would use this Mondo term https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0005113&viewMode=All&siblings=false
